src/main/java/com/github/onsdigital/babbage/api/endpoint/content/Hash.java

```diff
@@ -5,6 +5,7 @@
 import com.github.onsdigital.babbage.content.client.ContentReadException;
 import com.github.onsdigital.babbage.content.client.ContentResponse;
 import com.github.onsdigital.babbage.error.BabbageException;
+import com.github.onsdigital.babbage.response.util.CacheControlHelper;
```

Comment:
Do you still think that 'CacheControlHelper' is a good name for this class?

Reply:
Good point. I've renamed it ContentHashHelper now as discussed 👍